.github/workflows/test_rust.yml

```diff
@@ -94,3 +91,23 @@ jobs:
         with:
           name: swf_images
           path: tests/**/*.png.updated
+
+  check-required:
+    needs: changes
+    if: needs.changes.outputs.src == 'false'
+    name: Test Rust ${{ matrix.rust_version }} / ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.rust_version == 'nightly' || matrix.rust_version == 'beta' }}
```

Comment:
You don't really need `continue-on-error` and `fail-fast` because this dummy job cannot fail.